CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+# Contributing to the Opendatahub Operator
+
+Thanks for your interest in this project. You can contribute to this project in many different ways.
+
+## Issues and Enhancements
```

Comment:
I have added `component/<component-name>` labels to the repo. Can we add a line about adding appropriate labels when creating the issues.

Reply:
updated.